src/util/util.jl

```diff
@@ -36,8 +36,9 @@ function name(t::InteractionTerm)
     n
 end
 function name(t::FunctionTerm)
-    string(t.args_parsed[1].sym)
-end
-function name(t::FixedEffectTerm)
-    string(t.x)
-end
+    if t.f === FixedEffectModels.fe
+        string(t.exorig.args[end])
+    else
+        StatsBase.coefnames(t)
+    end
+end
```

Comment:
Sorry to bother you @kleinschmidt and @matthieugomez 

In a regression table we want to treat `FixedEffectTerm` differently from other `FunctionTerms`. 

E.g., for the term `fe(x)` we want to get `"x"`, while for all other functions `log(x)` we want to get `"log(x)"`.

It seems now that we cannot dispatch on `FixedEffectTerm` anymore. This code works, but I was wondering if there is a better way to handle this?

Reply:
yeah that's basically what I'd do.